### ISSUE TITLE
Set border style of table cell

### ISSUE THREAD

**Issue by kristofferjalen** [QuestPDF/QuestPDF#337]
I would like to set a border style to any of these values: `dotted`, `solid`, `double`, `dashed`.

**Comment by grinay** [comment]
Also need rounded border.  For now the only solution i see is a background image simulating such border.

**Comment by MarcinZiabek** [comment]
Hi 😁 You can also use the `Canvas` element to draw custom shapes around your content. Please take a look at this example: https://www.questpdf.com/api-reference/canvas.html

**Comment by grinay** [comment]
@MarcinZiabek yes, found already, thanks. BTW this page was in state 404 yesterday.

**Comment by MarcinZiabek** [comment]
The official documentation has been recently rewritten. For updated information related to this requirement, please check:
- [Complex Graphics](https://www.questpdf.com/api-reference/complex-graphics.html)
- [SkiaSharp Integration](https://www.questpdf.com/api-reference/skiasharp-integration.html).

Since there has been no recent activity, I’m closing this issue. If you experience a similar problem or have a related request, feel free to open a new GitHub issue. I appreciate your feedback! ❤️
